### ISSUE TITLE
Blurring an empty TextField (without doing anything in the field) causes a server visit

### ISSUE THREAD

**Issue by mstahv** [vaadin/flow#17429]
### Description of the bug

Blurring an empty TextField (without doing anything in the field) causes a server visit

### Expected behavior

There should be no obsolete server visits.

### Minimal reproducible example

Use this UI, click text field and document body repeatedly, watch the show in web inspectors network tab.

```
@Route
public class MainView extends VerticalLayout {

    public MainView() {
        TextField textField = new TextField();
        add(textField);
    }
}

```

### Versions

Vaadin: 24.1.4
Flow: 24.1.5
Java: Eclipse Adoptium 17.0.2
OS: aarch64 Mac OS X 13.5
Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36
Live reload: Java active (Spring Boot Devtools): Front end active

**Comment by knoobie** [comment]
Doesn't the dirty flag also fixes this? Or does this still happen with the new flag? https://github.com/vaadin/web-components/pull/6160

**Comment by TatuLund** [comment]
Related to https://github.com/vaadin/flow-components/issues/4986

**Comment by caalador** [comment]
As the event is a `validated` event let's see what happens after the components validation logic update.

**Comment by vursen** [comment]
The issue will be resolved as part of https://github.com/vaadin/flow-components/issues/5537.

**Comment by yuriy-fix** [comment]
The issue is resolved as a part of #5537. Fix is available in `24.3.0-beta1`.
`DateTimePicker` behaviour will be tackled separately.
